loleaflet/src/control/Control.LokDialog.js

```diff
@@ -1642,9 +1642,22 @@ L.Control.LokDialog = L.Control.extend({
 		}
 
 		var wrapper = L.DomUtil.get('sidebar-dock-wrapper');
-		if (wrapper && wrapper.offsetWidth)
-			this._map.options.documentContainer.style.right = wrapper.offsetWidth + 'px';
-		else
+		if (wrapper) {
+			var offsetWidth;
+			var panel = L.DomUtil.get('sidebar-panel');
+
+			if (panel.style.display !== 'none')
+				offsetWidth = wrapper.offsetWidth;
+			else {
+				panel.style.visibility = 'hidden';
```

Comment:
I don't understand something. If panel has `display == none`, shouldn't be the final result always equal to ~0px? do we need to calculate that value?

Reply:
> I don't understand something. If panel has display == none, shouldn't be the final result always equal to ~0px? do we need to calculate that value?


Setting the hidden visibility we force the browser to calculate the layout panel so the offsetWidth is not 0px.